client/components/cart.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+export const Cart = ({ order, tblWidthStyle1, tblWidthStyle2 }) => {
+  return (
+    <div className="cart">
+      <div id="cartTable">
+        <table>
+          <colgroup>
+            <col span="1" style={tblWidthStyle1} />
+            <col span="1" style={tblWidthStyle2} />
+            <col span="1" style={tblWidthStyle1} />
+            <col span="1" style={tblWidthStyle1} />
+          </colgroup>
+          <thead>
+            <tr>
+              <th><span className="text-leftalign">Image</span></th>
+              <th><span className="text-centeralign">Product</span></th>
+              <th><span className="text-leftalign">Price</span></th>
+              <th><span className="text-rightalign">Quantity</span></th>
+            </tr>
+          </thead>
+          <tbody>
+            {order.products && order.products.map(product => (
+              <tr key={product.id}>
+                <td><img src={product.imageUrl} width="100" height="100" /></td>
+                <td>
+                  <span>{product.name}</span>
+                  <br />
+                  <span>Shipping Firm: Silikon Road LLC</span>
```

Comment:
Have we added any/ are we going to add any functionality for showing whether a product is from a user vs. us?

Reply:
It looks like we'd have to dispatch a separate thunk/request to `GET user.name` where the id = `userId` on the product in the cart. We can get rid of this line for now, at the very least it shouldn't be hardcoded in there.